lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -0,0 +1,47 @@
+<div class="create-team wrapper-msg is-incontext urgency-low warning is-hidden" tabindex="-1">
+    <div class="msg">
+        <div class="msg-content">
+            <h3 class="title"><%- gettext("Your team could not be created!") %></h3>
+            <span class="screen-reader-message sr"></span>
+            <div class="copy">
+                <p></p>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="form-instructions create-team-instructions">
+    <p class="copy">
+        <%- gettext("Enter information to describe your team. You cannot change these details after you create the team.") %></p>
+</div>
+
+<div class="team-edit-fields">
+    <div class="team-required-fields">
+    </div>
+
+    <div class="team-optional-fields">
+    </div>
+</div>
+
+<div class="create-team form-actions">
+    <button class="action action-primary">
+        <%=
```

Comment:
Please replace these with `<%-` so that we can HTML escape the result of `gettext()`